docs/src/equations.md

```diff
@@ -111,25 +111,26 @@ The ``(2 \boldsymbol{\Omega}^v + \nabla_h \times \boldsymbol{u}_h) \times \bolds
 ```
 and the ``\frac{1}{\rho} \nabla_h p  + \nabla_h (\Phi + K)`` as
 ```math
-\frac{1}{\rho} G_h[p] + G_h[\Phi + K]
+\frac{1}{\rho} G_h[p] + G_h[\Phi + K] ,
 ```
+where all these terms are treated explicitly.
 
 #### Vertical velocity
 Similarly for vertical velocity
 ```math
 \frac{\partial}{\partial t} \boldsymbol{u}_v  =
   - (\nabla_v \times \boldsymbol{u}_h + \nabla_h \times \boldsymbol{u}_v) \times \boldsymbol{u}^h
-  - \frac{1}{\rho} \nabla_v p - \nabla_v(\Phi + K)
+  - \frac{1}{\rho} \nabla_v p - \nabla_v(\Phi + K) .
 ```
 The ``(\nabla_v \times \boldsymbol{u}_h + \nabla_h \times \boldsymbol{u}_v) \times \boldsymbol{u}^h`` term is discretized as
 ```math
-(C^f_v[\boldsymbol{u}_h] + C_h[\boldsymbol{u}_v]) \times I^f(\boldsymbol{u}_h)
+(C^f_v[\boldsymbol{u}_h] + C_h[\boldsymbol{u}_v]) \times I^f(\boldsymbol{u}_h) ,
 ```
 and the ``\frac{1}{\rho} \nabla_v p + \nabla_v(\Phi + K)`` term as
 ```math
-\frac{1}{I^f(\rho)} G^f_v[p] + G^f[K + \Phi]
+\frac{1}{I^f(\rho)} G^f_v[p] + G^f[K + \Phi] ,
 ```
-and treated implicitly.
+with the latter treated implicitly.
```

Comment:
@simonbyrne can you please confirm that I interpreted correctly? They way it read before was a bit confusing to understand which (only one or both terms?) are treated implicitly in this sentence. Thanks!